src/qt/diagnosticsdialog.cpp

```diff
@@ -193,21 +193,25 @@ void DiagnosticsDialog::on_testBtn_clicked() {
     else
         ui->boincPathResultLbl->setText("Failed");
     //find cpid
-    ui->findCPIDReaultLbl->setText("Testing...");
+#ifndef WIN32
+    ui->findCPIDResultLbl->setText("N/A");
+#else
+    ui->findCPIDResultLbl->setText("Testing...");
     this->repaint();
     result = DiagnosticsDialog::FindCPID();
     if(result == 1)
-        ui->findCPIDReaultLbl->setText(QString::fromStdString("Passed CPID: " + GetArgument("cpid", "")));
+        ui->findCPIDResultLbl->setText(QString::fromStdString("Passed CPID: " + GetArgument("PrimaryCPID", "")));
     else
-        ui->findCPIDReaultLbl->setText("Failed (Is CPID in gridcoinresearch.conf?)");
+        ui->findCPIDResultLbl->setText("Failed (Is PrimaryCPID in gridcoinresearch.conf?)");
+#endif
     //cpid valid
     ui->verifyCPIDValidResultLbl->setText("Testing...");
     this->repaint();
     result = DiagnosticsDialog::VerifyIsCPIDValid();
     if(result == 1)
         ui->verifyCPIDValidResultLbl->setText("Passed");
     else
-        ui->verifyCPIDValidResultLbl->setText("Failed (BOINC CPID and gridcoinresearch.conf CPID do not match)");
+        ui->verifyCPIDValidResultLbl->setText("Failed (BOINC CPID and gridcoinresearch.conf msPrimaryCPID do not match)");
```

Comment:
Change msPrimaryCPID to CPID in the texts. It's an implementation detail and makes the text a little reasier to read for the users.